.eslintrc

```diff
@@ -0,0 +1,4 @@
+extends: fxa/server
+
+rules:
+  handle-callback-err: 0
```

Comment:
Sshhh, I'm hoping nobody notices [since YAML is far superior to JSON &mdash; and I save an amazing 10 keystrokes!]